### PR TITLE
Add shoot-oidc-service extension to list

### PR DESCRIPTION
**Pull request by voelzmo**
**How to categorize this PR?**

/area documentation
/kind enhancement

**What this PR does / why we need it**:
Add shoot-oidc-service extension to list

**Release note**:

```other operator
NONE
```

### CODE REVIEW

File: extensions/README.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ Check out these repositories for implementations of the Gardener Extension contr
 
 - [Shoot Certificate Service](https://github.com/gardener/gardener-extension-shoot-cert-service)
 - [Shoot DNS Service](https://github.com/gardener/gardener-extension-shoot-dns-service)
+- [Shoot Openid Connect Service](https://github.com/gardener/gardener-extension-shoot-oidc-service)
```
Comment (dimityrmirchev) [inline extensions/README.md]:
```suggestion
- [Shoot OpenID Connect Service](https://github.com/gardener/gardener-extension-shoot-oidc-service)
```

**Review by dimityrmirchev — COMMENTED** [review]
Thanks! Just one minor suggestion.

File: extensions/README.md (hunk 1 above)
Comment (voelzmo) [reply extensions/README.md]: Thanks!

**Review by rfranzke — COMMENTED** [review]
/lgtm
/approve

**Review by dimityrmirchev — APPROVED** [review]

**Pull request merged**